src/encoded/item_utils/assay.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any, Dict
+
+
+def get_valid_sequencers(properties: Dict[str, Any]) -> str:
```

Comment:
I'm not familiar with python type hints yet but should the return hint be list[str] or something like that?  Just curious - I know it won't affect the function.